src/validations/bin/compile-sch.sh

```diff
@@ -15,13 +16,15 @@ java -cp "${SAXON_CP}" net.sf.saxon.Transform \
 
 echo "Preprocessing stage 2/3 of ${SOURCE}..."
 STAGE_2=$(mktemp)
+# shellcheck disable=SC2086
 java -cp "${SAXON_CP}" net.sf.saxon.Transform \
```

Comment:
it's the `${SAXON_OPTS}` at the end of that extended line.  We can't put it in quotes because we depend on the shell splitting the value into multiple arguments.

Reply:
formerly lines 14, 22, and 29; now 15, 24, and 32